src/util/walletUtils.ts

```diff
@@ -190,7 +190,47 @@ export class WalletUtils {
             }
         });
     }
+//subject to change 
```

Comment:
both are doing the same thing, why not just resolve the accountInfo, then get the property after resolve